src/clp_logging/handlers.py

```diff
@@ -302,6 +312,21 @@ def update(self, loglevel: int, log_timestamp_ms: int, log_fn: Callable[[str], N
         self.soft_timeout_thread.start()
 
 
+def _get_mutex_context_from_loglevel_timeout(loglevel_timeout: Optional[CLPLogLevelTimeout]) -> Any:
+    """
+    Gets a mutual exclusive context manager for IR stream access.
+
+    The return type should be `AbstractContextManager[Optional[bool]]`, but it
+    is annotated as `Any` for backward compatibility, since the generic
+    `AbstractContextManager` is not available before Python 3.9 (#18239).
+
```

Comment:
```suggestion
    NOTE: The return type should be `AbstractContextManager[Optional[bool]]`,
    but it is annotated as `Any` to satisfy the linter in Python 3.7 and 3.8,
    as `AbstractContextManager` was introduced in Python 3.9 (#18239).

```
We usually use `NOTE: xxx` for this type of comments